tests/test_pl.py

```diff
@@ -64,6 +68,22 @@ def test_cardinal(self):
             "osiemdziesiąt cztery miliony dwieście dwadzieścia "
             "tysięcy dwieście dziewięćdzisiąt jeden"
         )
+        self.assertEqual(
+            num2words(
+                963301000001918264129471042047146102350812074235000101020000120324,
```

Comment:
Try to do something like this in order to be compliant with PEP8 
```suggestion
n = 963301000001918264129471042047146102350812074235000101020000120324
self.assertEqual(
    num2words(ln, lang=pl)
```

Reply:
I've shortened the number and multiplied by power of 10 instead. Smaller parts are already covered, so this specific test checks only large numbers' (>10**30) names.